src/services/accounting.js

```diff
@@ -3,8 +3,8 @@
  */
 export const Accounting = {
   create: ({ serviceURL }) => ({
-    record: async (cid, options) => {
-      console.log(`using ${serviceURL} to record a GET for ${cid} with options`, options)
+    record: async (cid, bytes, servedAt) => {
```

Comment:
I will create another PR to integrate with `w3up-client` and execute the `usage/record` invocation from here.